src/web/islands/islands.tsx

```diff
@@ -96,13 +100,18 @@ export const hydrateIslands = (
             root: 'share-count',
         },
         {
-            component: MostViewed,
+            component: MostViewedGrid,
             props: {
                 config,
                 pillar,
                 sectionName,
             },
-            root: 'most-viewed',
+            root: 'most-viewed-grid',
+        },
+        {
+            component: MostViewedList,
```

Comment:
I think we should prioritise this list view over the grid (and maybe over things like share count). What do you think?

Reply:
I agree. I hadn't given the priority much thought before. I've moved it before 'share-count' and after 'edition-root' in low priority islands.

Reply:
Perfect